engine/client/client.go

```diff
@@ -288,10 +286,7 @@ func (c *Client) startEngine(ctx context.Context) (rerr error) {
 
 	c.bkClient = bkClient
 
-	if c.EngineNameCallback != nil {
-		engineName := fmt.Sprintf("%s (version %s)", bkInfo.BuildkitVersion.Revision, bkInfo.BuildkitVersion.Version)
-		c.EngineNameCallback(engineName)
-	}
+	slog.Info("Connected to engine", "name", bkInfo.BuildkitVersion.Revision, "version", bkInfo.BuildkitVersion.Version)
```

Comment:
Worth a shot yea! We don't normally show logs so this was probably just flying under the radar.

There's a general idea that the bottom of the screen in a CLI is actually the "prime real estate" unlike with a web UI, since that's where the user will be looking - but I think you could easily make the argument that the active things belong in the prime real estate, instead of the logs, which are more in the background.

Reply:
Updated to do this now, it's thankfully a super simple change - it also improves consistency, since this is what we do during `finalRender` already :tada: